production/template.yml

```diff
@@ -0,0 +1,321 @@
+#
+# Copyright 2017 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+apiVersion: v1
+kind: Template
+labels:
+  template: jaeger-template
+  jaeger-infra: template
+message: 'A Jaeger service has been created in your project.'
+metadata:
+  name: jaeger-template
+  annotations:
+    description: Jaeger Distributed Tracing Server
+    iconClass: icon-cogs
+    openshift.io/display-name: Jaeger
+    tags: instant-app,apm,opentracing
+  labels:
+    name: jaeger-infra
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: "${NAMESPACE}"
+    name: cassandra
+    labels:
+      name: jaeger-cassandra-service
+      jaeger-infra: cassandra-service
+  spec:
+    ports:
+    - port: 7000
+      name: intra-node
+    - port: 7001
+      name: tls-intra-node
+    - port: 7199
+      name: jmx
+    - port: 9042
+      name: cql
+    clusterIP: None
+    selector:
+      app: cassandra
+- apiVersion: "apps/v1beta1"
+  kind: StatefulSet
+  metadata:
+    namespace: "${NAMESPACE}"
+    name: cassandra
+    labels:
+      name: jaeger-cassandra-statefulset
+      jaeger-infra: cassandra-statefulset
+  spec:
+    serviceName: cassandra
+    replicas: 2
+    template:
+      metadata:
+        labels:
+          app: cassandra
+      spec:
+        containers:
+        - name: cassandra
+          image: jpkroehling/cassandra
```

Comment:
We discussed that at the beginning, IIRC. This is meant to be temporary, as we have another task about using the same Cassandra for all Hawkular components. I added a issue, to get this sorted out.